support-frontend/assets/helpers/user/user.ts

```diff
@@ -45,6 +41,14 @@ function isTestUser(): boolean {
 	return isDefined(uatMode) || isDefined(testCookie);
 }
 
+function getUserStateField(): string | undefined {
+	const user = getGlobal<User>('user');
+	if (user) {
+		return user.address4 ?? window.guardian.geoip?.stateCode;
+	}
+	return window.guardian.geoip?.stateCode;
```

Comment:
Is there a difference between this and the following? It looks to me like the should be the same, so the shorter form seems preferable, but I’m not very confident in my typescript.

```suggestion
	return user?.address4 ?? window.guardian.geoip?.stateCode;
```